lib/rust/profiler/src/build.rs

```diff
@@ -0,0 +1,121 @@
+//! Supports constructing a profile file.
```

Comment:
This docs does not really explain why this file is needed and what it does. Why the custom build.rs is needed? An explanation of that should be provided here for the reader.

Reply:
`build` was a bad name for a module because it looks like a build script. I have renamed it and documented it a little more.